stanford-earth/SE3_BLT
# Updates

# READY FOR REVIEW

# Summary
- Re-syncs config with staging and local changes.

# Needed By (Date)
- Sooner the better.

# Urgency
- Moderate

# Steps to Test
1. Check out this branch
2. Refresh your environment (code & db)
3. If you used blt great, if you did so manually be sure to import the default and local config and then features
4. Clear caches and review changes.

# Affected Projects or Products
- Just Earth

# Associated Issues and/or People
- None.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
